README.md

```diff
@@ -48,6 +48,7 @@ It collect key metrics about:
 | rds_exporter_errors_total | | Total number of errors encountered by the exporter |
 | rds_free_storage_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Free storage on the instance |
 | rds_freeable_memory_bytes | `aws_account_id`, `aws_region`, `dbidentifier` | Amount of available random access memory. For MariaDB, MySQL, Oracle, and PostgreSQL DB instances, this metric reports the value of the MemAvailable field of /proc/meminfo |
+| rds_instance_age_seconds | `aws_account_id`, `aws_region`, `dbidentifier` | Instance age |
```

Comment:
```suggestion
| rds_instance_age_seconds | `aws_account_id`, `aws_region`, `dbidentifier` | Time since instance creation |
```
or something to avoid confusing it with uptime (which I agree, can be quite different).

Reply:
I applied the change (and reflected it in metric help in `exporter.go`).

Note: So far, I didn't see metrics in RDS API or Cloudwatch API to collect the uptime metric. In our context, we have it using PostgreSQL exporter.